src/main/resources/migrations/00008_create_products_gateway_account_idx.sql

```diff
@@ -0,0 +1,6 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:add_index_to_gateway_account_id_on_products
+CREATE INDEX gateway_account_idx ON products(gateway_account_id);
+
+--rollback alter table products drop column gateway_account_id
```

Comment:
rollback should probably drop the index and not the column

Reply:
Spot on, thank you for noticing it!